src/StoreApi/Routes/V1/AbstractCartRoute.php

```diff
@@ -86,11 +97,11 @@ public function get_response( \WP_REST_Request $request ) {
 
 		if ( is_wp_error( $response ) ) {
 			$response = $this->error_to_response( $response );
-		} elseif ( in_array( $request->get_method(), [ 'POST', 'PUT', 'PATCH', 'DELETE' ], true ) ) {
+		} elseif ( $this->is_update_request( $request ) ) {
 			$this->cart_updated( $request );
 		}
```

Comment:
Just a really tiny nitpick, we should do like before and return early on wp_error and ditch the elseif. Having the success flow on the function's main branch will never require else/ifelse, and those can always be avoided.
```
if ( is_wp_error( $response ) ) {
    return $this->error_to_response( $response );
}

if ( $this->is_update_request( $request ) ) {
   $this->cart_updated( $request );
}
```